### PR TITLE
docs(faq): summarize difference between deis and deisctl clients

### LINKED ISSUE #3401: proposal renaming of the clients
deisctl /deis does not realy reflect what they are doing

deis_server_ctl / deis_ops_ctl
deis_client_ctl / deis_dev_ctl

would be more clear,
even if more typing is involved.

of course thats just semantics but for new guys trying deis for the first time .. its all about understanding

### PR DESCRIPTION
**Pull request by mboersma**
Closes #3401.

### CODE REVIEW

File: docs/installing_deis/install-deisctl.rst [hunk 1]
```diff
@@ -6,8 +6,7 @@
 Install deisctl
 ===============
 
-The Deis Control Utility, or ``deisctl`` for short, is a command-line client used to configure and
```
Comment (bacongobbler) [inline docs/installing_deis/install-deisctl.rst]: IMO I liked the older description as it was more verbose about `deisctl`

File: docs/using_deis/install-client.rst [hunk 2]
```diff
@@ -5,8 +5,9 @@
 
 Install the Client
 ==================
-The Deis command-line interface (CLI), or client, allows you to interact
```
Comment (bacongobbler) [inline docs/using_deis/install-client.rst]: Same reason as above; just seems more formal to me

**Comment by bacongobbler** [comment]
these are personal nitpicks so it's not really a hard requirement to amend the docs to my liking. LGTM :)

**Comment by carmstrong** [comment]
:ship:

**Pull request merged**